stdlib/FileWatching/src/FileWatching.jl

```diff
@@ -774,6 +850,10 @@ giving the result of watching the file.
 
 This behavior of this function varies slightly across platforms. See
 <https://nodejs.org/api/fs.html#fs_caveats> for more detailed information.
+
+This is a thin wrapper over calling `wait` on a [`FileMonitor`](@ref), which implements
+the functionality, but this function has a small race window between consecutive calls to
+`watch_file` where the file might change without being detected.
```

Comment:
```suggestion
This is a thin wrapper over calling `wait` on a [`FileMonitor`](@ref). This function has a small race window between consecutive calls to
`watch_file` where the file might change without being detected. To avoid this race, use

    fm = FileMonitor(path)
    wait(fm)

directly, re-using the same `fm` each time you `wait`.